apps/desktop/electron/core/config.ts

```
@@ -0,0 +1,256 @@
+  if (id === "auto") return "Auto (Smart Router)";
+  return id.split("/").at(-1)?.replaceAll("-", " ") ?? id;
+}
+
+function parseJsonObject(raw: string, label: string): Record<string, any> {
+  if (!raw.trim()) return {};
+  try {
+    const value = JSON.parse(raw) as unknown;
+    if (!isJsonObject(value)) throw new Error("the root value must be an object");
+    return value;
+  } catch (error) {
+    throw new Error(`Invalid ${label}: ${error instanceof Error ? error.message : String(error)}`);
```

ESLint

[error] There is no `cause` attached to the symptom error being thrown.

(preserve-caught-error)